src/timestamp.js

```diff
@@ -1,4 +1,6 @@
 export default function secondsToTimestamp(seconds) {    //formats seconds into mm:ss if less than an hour, hh:mm:ss if greater than an hour
-    const timeString = new Date(seconds * 1000).toISOString();
-    return seconds >= 3600 ? timeString.substr(11, 8) : timeString.substr(14, 5);
+    const posSeconds = Math.abs(seconds);
+    const timeString = new Date(posSeconds * 1000).toISOString();
+    const subTimeString = posSeconds >= 3600 ? timeString.substr(11, 8) : timeString.substr(14, 5);
+    return seconds < 0 ? "-" + subTimeString : subTimeString;
```

Comment:
This is a little repetitive, maybe just `(seconds < 0 ? "-" : "") + subTimeString`